backend/packages/Upgrade/src/api/services/FeatureFlagService.ts

```diff
@@ -107,7 +123,9 @@ export class FeatureFlagService {
   ): Promise<FeatureFlag[]> {
     logger.info({ message: 'Find paginated Feature flags' });
 
-    let queryBuilder = this.featureFlagRepository.createQueryBuilder('feature_flag');
+    let queryBuilder = this.featureFlagRepository
+      .createQueryBuilder('feature_flag')
+      .leftJoinAndSelect('feature_flag.featureFlagExposures', 'featureFlagExposures');
```

Comment:
This is too much data to query in production. You are only interested in the count of the exposure. Have a separate query to get that count.